### PR TITLE
Update ITK 4.13.1+

### PR DESCRIPTION
**Pull request by dzenanz**

### CODE REVIEW

File: Base/QTApp/qSlicerApplicationHelper.cxx [hunk 1]
```diff
@@ -66,6 +66,10 @@
 # include <PythonQtPythonInclude.h>
 #endif
 
+#ifdef _WIN32
+#include <Windows.h> //for SetProcessDPIAware
```
Comment (lassoan) [inline Base/QTApp/qSlicerApplicationHelper.cxx]: We've been building Slicer successfully without this on many computers. Do you think it's related to ITK update?

Comment (dzenanz) [reply Base/QTApp/qSlicerApplicationHelper.cxx]: I doubt it is related to ITK update. More likely it is some combination of Qt version, Windows SDK version etc.

Comment (jcfr) [reply Base/QTApp/qSlicerApplicationHelper.cxx]: `_WIN32` -> `Q_CC_MSVC` ? 

Let's integrate this separately

Comment (dzenanz) [reply Base/QTApp/qSlicerApplicationHelper.cxx]: Sure. Should I make a new PR? Or just merging the first commit is possible?

**Comment by jcfr** [comment]
> Or just merging the first commit is possible?

I am taking care of the first commit, local build completed. Now integrating

**Comment by jcfr** [comment]
> a new PR?

To track the other change about DPI, that would be great

**Comment by jcfr** [comment]
Closing. Integrated in [r27399](http://viewvc.slicer.org/viewvc.cgi/Slicer4?view=revision&revision=27399)

If dashboard behaves, we will then integrate https://github.com/Slicer/Slicer/pull/1014

**Pull request closed without merging**